### PR TITLE
Jbtm 3699 jf

### PR DESCRIPTION
**Pull request by jmfinelli**
Fixed `XTS` and `compensating-transactions`

### CODE REVIEW

File: compensating-transactions/mongodb-simple/pom.xml [hunk 1]
```diff
@@ -115,7 +83,7 @@
             <exclusions>
                 <exclusion>
                     <groupId>org.jboss.logging</groupId>
-                    <artifactId>jboss-logging-spi</artifactId>
+                    <artifactId>jboss-logging-spi-</artifactId>
```
Comment (marcosgopen) [inline compensating-transactions/mongodb-simple/pom.xml]: typo

Comment (jmfinelli) [reply compensating-transactions/mongodb-simple/pom.xml]: That was hard to spot! Thanks @marcosgopen

**Review by mmusgrov — APPROVED** [review]
Very good PR, and in amongst the formatting etc I noticed some good fixes.

**Comment by jmfinelli** [comment]
Thanks @mmusgrov for your review! Really appreciated!

@marcosgopen Thanks for your review. I have addressed the typo (I think).

**Comment by mmusgrov** [comment]
@jmfinelli can this one be merged yet

File: XTS/wsat-jta-multi_service/src/test/resources/arquillian.xml [hunk 2]
```diff
@@ -20,35 +20,35 @@
...
-	</group>
+        <container qualifier="jboss2">
+            <configuration>
+                <property name="jbossHome">${jboss2.home:target/jboss2}</property>
+                <property name="javaVmArguments">${modular.jdk.args} ${server.jvm.args}
+                    -Djboss.socket.binding.port-offset=100
+                </property>
+                <property name="serverConfig">standalone-xts.xml</property>
+                <property name="managementAddress">${node.address}</property>
+                <property name="managementPort">10090
```
Comment (marcosgopen) [inline XTS/wsat-jta-multi_service/src/test/resources/arquillian.xml:47]: Please @jmfinelli can you remove the new line at line 47? I got NumberFormatException with the new line.

**Review by marcosgopen — COMMENTED** [review]
Thanks for you PR Manuel!

File: XTS/wsat-jta-multi_service/src/test/java/org/jboss/narayana/quickstarts/wsat/jtabridge/fromjta/BridgeFromJTATest.java [hunk 3]
```diff
@@ -96,8 +93,12 @@ public static JavaArchive createTestArchive2() {
     public void setupTest() throws Exception {
         Context initialContext = new InitialContext();
         ut = (UserTransaction)initialContext.lookup("java:comp/UserTransaction");
-        firstClient = FirstClient.newInstance();
-        secondClient = SecondClient.newInstance();
+        try {
+            firstClient = FirstClient.newInstance();
+            secondClient = SecondClient.newInstance();
```
[the remark this answers is not in the document]
Comment (jmfinelli) [reply XTS/wsat-jta-multi_service/src/test/java/org/jboss/narayana/quickstarts/wsat/jtabridge/fromjta/BridgeFromJTATest.java:98]: Yeah, that's a leftover from a test :sweat_smile: Thank you Marco

File: XTS/wsat-jta-multi_service/src/test/resources/arquillian.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jmfinelli) [reply XTS/wsat-jta-multi_service/src/test/resources/arquillian.xml]: Thanks Marco!

**Comment by marcosgopen** [comment]
Thank you @jmfinelli for addressing all my comments!

**Review by marcosgopen — APPROVED** [review]
Thanks for your PR @jmfinelli !

**Pull request merged**